src/chapters/loops/iterating-over-collections.rst

```diff
@@ -0,0 +1,86 @@
+Iterating Over Collections
+==========================
+
+One of the most common uses of a ``for`` loop is to carry out a task once for each item in a collection. We have learned about two types of collections, strings and arrays. When using a loop with a collection in this way, we say that the loop *iterates over* the collection.
```

Comment:
Bold or italics for “collections”, “strings” and “arrays”.

Reply:
For emphasis? Or because they are vocab terms? If you mean the latter, then these terms have already defined in earlier sections, hence don't need to be bold.

If you mean for emphasis, I don't agree. The point seem to warrant it, and having four italicized words in a single paragraph should rarely, if ever, happen.

Reply:
Emphasized because they are vocab words.  Since they have already been defined, then the point I was making has been addressed.